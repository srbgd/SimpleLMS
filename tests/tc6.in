

drop
register student --login p --password pass --name test --address innopolis --phone-number 88005553535 --card-number 9876
insert book --title a title --authors unknown --publisher empty --edition none --date unknown --price 1000
copy 1
copy 1
login
p
pass
checkout 1
checkout 1
exit
