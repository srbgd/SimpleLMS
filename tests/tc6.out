Success
True
True
True
True
True
Success
True
False
